pkgs/by-name/fu/fuzzel/package.nix

```
@@ -13,14 +13,16 @@
 , tllist
 , fcft
 , enableCairo ? true
-, svgSupport ? true
 , pngSupport ? true
-# Optional dependencies
+, svgSupport ? true
+, svgBackend ? "nanosvg" # alternative: "librsvg"
+  # Optional dependencies
 , cairo
 , libpng
+, librsvg
 }:
 
-assert svgSupport -> enableCairo;
+assert (svgBackend == "nanosvg") -> enableCairo;
```

```suggestion
assert (svgSupport && svgBackend == "nanosvg") -> enableCairo;
```

This should be properly guarded, as if `svg-backend` is `"none"`, we don't actually need to enable Cairo

---

Fixed